content/applications/hr/recruitment/new_job.rst

```diff
@@ -99,27 +108,32 @@ An *Interview Form* is used to determine if a candidate is a good fit for a job
 forms can be as specific or general as desired, and can take the form of a certification, an exam,
 or a general questionnaire. Interview forms are determined by the recruitment team.
 
-All interview forms must be created, there are no pre-configured forms in Odoo. To create an
-interview form, start from the recruitment tab of the :guilabel:`Job Position` form. In the
-:guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
-several options populate beneath the entry, :guilabel:`Create (interview form name)`,
-:guilabel:`View all`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`. and
-a :guilabel:`Create Interview Form` pop-up window appears.
+Before creating an interview form, ensure the proper settings are enabled. Navigate to
+:menuselection:`Recruitment app --> Configuration --> Settings`, and under the
+:guilabel:`Recruitment Process` section, ensure the :guilabel:`Send Interview Survey` option is
+enabled.
+
+Since there are no pre-configured forms in Odoo, all interview forms must be created. To create an
+interview form, start from the :guilabel:`Recruitment` tab of the :guilabel:`Job Position` form. In
+the :guilabel:`Interview Form` field, enter a name for the new interview form. As the name is typed,
+several options populate beneath the entry: :guilabel:`Create (interview form name)`,
+:guilabel:`Search More...`, and :guilabel:`Create and edit...`. Click :guilabel:`Create and edit...`
+and a :guilabel:`Create Interview Form` pop-up window appears.
 
 .. note::
-   The option :guilabel:`View all` only appears if there are any interview forms already created.
-   If no interview forms exist, the only options available are :guilabel:`Create (interview form
-   name)`, and :guilabel:`Create and edit...`.
+   The option :guilabel:`Search More...` only appears if there are any interview forms already
+   created. If no interview forms exist, the only options available are :guilabel:`Create (interview
+   form name)`, and :guilabel:`Create and edit...`.
 
 First, enter a name for the form in the :guilabel:`Name` field. This should be indicative of when
```

Comment:
From here on down, i think you can shorten this doc a great deal by simply linking to the doc on creating surveys. I just rewrote it from scratch (it hasn't been published yet) and it covers every single element that you mentioned -- along with specific screenshots and elaborated descriptions.

So, you can simply allude to that (w/a link) with something like "...and then proceed to fill out this pop-up from the same way you would a typical survey..."

Reply:
OMG! Ok that does save about 445 lines or so 0_0

I assume it's the "survey essentials" document?